docs/src/pages/components/app-bar/app-bar.md

```diff
@@ -136,3 +136,27 @@ function HideOnScroll(props) {
   );
 }
 ```
+
+## Enable Color on Dark
+
+From v5 onward, `color` prop has no effect on dark mode according to [material design spec](https://material.io/design/color/dark-theme.html). You can opt out by passing `enableColorOnDark` prop.
```

Comment:
Prefer breaking lines between sentences, makes reviewing easier:

```suggestion
From v5 onward, `color` prop has no effect on dark mode according to [material design spec](https://material.io/design/color/dark-theme.html).
You can opt out by passing `enableColorOnDark` prop.
```

Reply:
We usually don't mention previous versions:

```suggestion
The `color` prop has no effect on dark mode according to [material design spec](https://material.io/design/color/dark-theme.html).
You can opt out by passing `enableColorOnDark` prop.
```

Reply:
@mbrookes Are you OK to settle on this terminology?

```suggestion
The `color` prop has no effect on dark mode according to [Material Design guidelines](https://material.io/design/color/dark-theme.html).
You can opt out by passing `enableColorOnDark` prop.
```

Reply:
> @mbrookes Are you OK to settle on this terminology?

No, but it seems that it was merged before you asked 🙂.

A gentle reminder to @mui-org/maintainers to tag me on any PRs with new or significantly changed language or terminology.

Reply:
@mbrookes You have another shoot at it in #26628.